### PR TITLE
Fixed ISSUE #10465 - Sentinel Central Workbook Patch

### PR DESCRIPTION
**Pull request by melatonein5**
Fixed ISSUE #10465  where incidents were not being counted correctly.

   Required items, please complete
   
   Change(s):
   - Changed KQL Queries to include an `arg_max()` to prevent incidents from being counted multiple times in Workbooks\Sentinel_Central.json
   - Updated Change Log in Workbooks\Sentinel_Central.json

   Reason for Change(s):
   - See ISSUE #10465 

   Version Updated:
   - Yes
   - Updated Changelog

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by melatonein5** [comment]
> @microsoft-github-policy-service agree [company="JMNetwork"]

**Comment by melatonein5** [comment]
@microsoft-github-policy-service agree

**Comment by melatonein5** [comment]
Pipeline claiming template failing at offset 200461 due to a resource reference. This is line 4088 , which is  `"fieldName": "Status",` which is obviously not a resource, unless I am missing something.

File: Workbooks/Sentinel_Central.json [hunk 1]
```diff
@@ -4248,6 +4211,9 @@
       "name": "group - rules"
     }
   ],
+  "fallbackResourceIds": [
```
Comment (v-shukore) [inline Workbooks/Sentinel_Central.json:4214]: Hi @melatonein5 could you please remove this block and try

**Comment by melatonein5** [comment]
@v-shukore block removed, checks passed.

**Review by v-shukore — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**